src/main/java/net/sf/jabref/bst/BibtexCaseChanger.java

```diff
@@ -129,17 +134,17 @@ private String doChangeCase(String s, FORMAT_MODE format) {
         return sb.toString();
     }
 
-    private int decrBraceLevel(String string, int braceLevel) {
-        if (braceLevel == 0) {
+    private int decrBraceLevel(String string, int bLevel) {
+        if (bLevel == 0) {
```

Comment:
I read the code and decided to inline the methods as they were only called once. This also lead to warnings messages which made more sense, removal of an unused argument in BibtexWidth and change of the corresponding test.